src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,7 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
+* Remove source_resource_id of rolebinding update command
```

Comment:
In fact, this is an inevitable breaking change caused by REST service. Therefore, it is recommended to mark it as a breaking change in the history notes and explain the reason
```suggestion
*  [BREAKING CHANGE] Since the service no longer supports updating source resource id for role binding, so remove --source-resource-id of `aks trustedaccess rolebinding update` command
```

Reply:
see comment below, this feature is in private preview and not accessible by external yet.